libr/core/cmd_anal.c

```diff
@@ -380,7 +380,6 @@ static const char *help_msg_afc[] = {
 	"Usage:", "afc[agl?]", "",
 	"afc", " convention", "Manually set calling convention for current function",
 	"afc", "", "Show Calling convention for the Current function",
-	"afc=", "([cctype])", "Select or show default calling convention",
```

Comment:
Description clearly says what `afc=` does

Reply:
Yep i know. But im suggesting to change the behaviour of the afc= command to do that. Which can be done after merging this 👍

Reply:
(it's just a suggestion)